### ISSUE TITLE
Destination Feature Count is Double the Source Count

### ISSUE THREAD

**Issue by stdavis** [agrc/forklift#337]
![image](https://user-images.githubusercontent.com/1326248/101933358-4f376e80-3b99-11eb-9feb-c990da1227df.png)

We've seen this before. Not sure what the issue is. Maybe the hashes get corrupted? Or maybe the deletes don't complete successfully? We should probably prioritize this issue since it seems to be a big deal to me.

**Comment by steveoh** [comment]
is this still an issue mr @stdavis?

**Comment by stdavis** [comment]
I don't ever remember addressing it. I'm not sure that I remember seeing these warnings recently. Have you?

**Comment by steveoh** [comment]
I've been using the slack report and only see that there are warnings but not what those warnings are.

**Comment by stdavis** [comment]
It doesn't appear that this has been an issue since Dec 2020
![image](https://user-images.githubusercontent.com/1326248/132536542-dff53bd4-8699-4925-8352-b18b9c3453af.png)

I've looked through the commits since then. Maybe this was solved with change detection? That was done in March. Or maybe it was some sort of data/database issue that was resolved? Either way, I think that we can close this for now. Perhaps we should add the warning messages to slack at some point. I don't look at the email reports either.
